### ISSUE TITLE
Change from polling to event based change detection for pipelines

### ISSUE THREAD

**Issue by jordan-evans** [awslabs/aws-service-catalog-factory#47]
We've just hit the below error in our account

```
ERROR MainThread CREATE_FAILED: Maximum number of pipelines using periodic source checks in account (60) exceeded. 
Switch to event-based change detection such as Amazon CloudWatch Events or webhooks to save this pipeline. 
Learn more: https://docs.aws.amazon.com/console/codepipeline/polling-to-events-based 
(Service: AWSCodePipeline; Status Code: 400; Error Code: LimitExceededException; Request ID: xxx)  
```

Can the Product Pipelines be switched to using CloudWatch Events for change detection?

**Comment by jordan-evans** [comment]
It looks like the pipelines should already be event based. We're investigating now to see if there's anything with our account causing this that's not related to SCF

**Comment by eamonnfaherty** [comment]
The pipelines created by factory do not assume PollForSourceChanges should be set to false.  Using the github source type you can set this but there is no option for setting it in codecommit sources.

**Comment by eamonnfaherty** [comment]
working on a fix now

**Comment by eamonnfaherty** [comment]
This is now fixed in release 0.1.36
